client_options.go

```diff
@@ -0,0 +1,16 @@
+package recurly
+
+// Region specific to Recurly
+type region string
```

Comment:
By not exposing the `region` type outside of the package, we can require that a developer makes use of one of the restricted const's below.